### ISSUE TITLE
The pod version is too old

### ISSUE THREAD

**Issue by Rain-dew** [MailCore/mailcore2#1976]
The library introduced via Pods is from 2020, why not use the latest Tag? Can you add one? Importing through the Project file is a bit more complicated

**Comment by WhyWake** [comment]
Agree, would be great if a new release is pushed with latest changes. I filed https://github.com/MailCore/mailcore2/issues/1985 specifically for a fix for CVE-2021-26911, but a release with all the latest changes would be great as well.
